Library/Homebrew/compat/md5.rb

```diff
@@ -1,26 +1,26 @@
 class Formula
   def self.md5(val)
+    odisabled "Formula.md5", "Formula.sha256"
     stable.md5(val)
   end
 end
 
 class SoftwareSpec
   def md5(val)
+    odisabled "SoftwareSpec#md5", "SoftwareSpec#sha256"
     @resource.md5(val)
   end
 end
 
 class Resource
   def md5(val)
+    odisabled "Resource#md5", "Resource#sha256"
     @checksum = Checksum.new(:md5, val)
   end
 end
```

Comment:
As I said, all of these can only be warning. The md5 support is already long gone because the below odie in `Pathname#md5`  

Also as for warning, I'm not sure it should be printed repeatedly.